worlds/zillion/item.py

```diff
@@ -1,6 +1,34 @@
+from typing import Counter
 from BaseClasses import Item, ItemClassification as IC
 from zilliandomizer.logic_components.items import Item as ZzItem
 
+_useful_thresholds = {
+    "Apple": 9999,
+    "Champ": 9999,
+    "JJ": 9999,
+    "Win": 9999,
+    "Empty": 0,
```

Comment:
Why are Empty and Bread included if the default is just 0?

Reply:
The values are here so that it's more clear when reading this code what happens with Empty and Bread.
If this `"Empty": 0,` weren't here, it would take a little more mental energy for the reader to figure out what happens with it.

The default is just a safety in case something changes in the future.